backend/app/src/main/java/ch/zhaw/studyflow/domain/calendar/CalendarDao.java

```diff
@@ -39,19 +39,12 @@ public interface CalendarDao {
      */
     Calendar update(Calendar calendar);
 
-    /**
-     * Gets the calendar ID.
-     *
-     * @param calendar the calendar
-     * @return the calendar ID
-     */
-    long getCalendarId(Calendar calendar);
 
     /**
-     * Sets the calendar ID.
+     * Retrieves all calendars.
      *
-     * @param calendar the calendar
-     * @param calendarId the new calendar ID
+     * @return a list of all calendars
      */
-    void setCalendarId(Calendar calendar, long calendarId);
+    List<Calendar> getAll();
```

Comment:
I don't think we need a method to get all calendars. Since this would return all calendars for all users.